synapseclient/core/cache.py

```diff
@@ -281,22 +280,43 @@ def _cache_dirs(self):
                     if os.path.isdir(path2) and re.match('\\d+', item2):
                         yield path2
 
-    def purge(self, before_date, dry_run=False):
+    def purge(self, before_date=None, after_date=None, dry_run=False):
         """
         Purge the cache. Use with caution. Delete files whose cache maps were last updated prior to the given date.
 
         Deletes .cacheMap files and files stored in the cache.cache_root_dir, but does not delete files stored outside
         the cache.
+
+        :param before_date: To specify the date that all the files in Synapse caches which creating date before it will
+                            be removed.
+        :param after_date:  To specify the date that all the files in Synapse caches which creating date after it will
+                            be removed.
+        :param dry_run:     To determine purge method actually remove the files. If dry_run is True then just print out
+                            the files that will be removed.
+
+        :returns: count how many file will be removed
```

Comment:
How about the following:
        Either the before_date or after_date must be specified. If both are passed, files between the two dates are selected for removal.

        :param before_date: if specified, all files before this date will be removed
        :param after_date:  if specified, all files after this date will be removed
        :param dry_run:     if dry_run is True, then the selected files are printed rather than removed
        :returns: the number of files selected for removal

Reply:
I think you missed part of this comment on that last commit. I was also suggesting that the :param: strings get updated in this comment, I think they not quite clear enough as is.